gui/src/components/Txs.tsx

```diff
@@ -1,12 +1,13 @@
 import { ArrowRightIcon } from "@heroicons/react/20/solid";
 import { listen } from "@tauri-apps/api/event";
 import classnames from "classnames";
-import { formatEther } from "ethers/lib/utils";
 import { useEffect } from "react";
 import useSWR from "swr";
 import truncateEthAddress from "truncate-eth-address";
+import { formatEther } from "viem";
+import type { TransactionReceipt } from "viem/src/types";
```

Comment:
You can't import from `viem/src` like that here. See `exports` in viem's package.json. This is causing the error that you reported in https://github.com/wagmi-dev/viem/issues/371